packages/components/psammead-story-promo/src/index.jsx

```diff
@@ -114,16 +121,30 @@ export const Summary = styled.p`
   }
 `;
 
-const StoryPromo = ({ image, info }) => (
+const StoryPromo = ({ image, info, mediaInfo }) => (
   <StoryPromoWrapper>
-    <ImageGridItem>{image}</ImageGridItem>
+    <ImageGridItem>
+      {image}
+      {mediaInfo ? (
+        <InlineMediaIndicator
+          datetime={mediaInfo.datetime}
```

Comment:
Rather than passing in a data object as a prop, could you pass a `InlineMediaIndicator` element? Would match how we handle images and info with the headline etc. E.g. rather than passing in headline text, we pass in `Info` which contains elements with that text inside 🙈

Using the `mediaInfo` object values here is very container like, and should we need to update what's passed to `InlineMediaIndicator`, it would require changing in that component, simorgh and here, rather than just the first two.

Just a suggestion, but if you export `InlineMediaIndicator`, it could maybe be passed in as part of the `image` prop, or leave `mediaInfo` and accept the react element?

Reply:
Good point 👍 Done now